test/e2e/tests/tokens/add-hide-token.spec.js

```diff
@@ -126,16 +127,16 @@ describe('Add existing token using search', function () {
           tag: 'p',
         });
         await driver.clickElement({ text: 'Next', tag: 'button' });
-        await driver.clickElement(
+        await driver.clickElementAndWaitToDisappear(
```

Comment:
this fixes the flakiness on the webpack build, where we proceed with opening the token details before the modal has been closed, causing an error